File: app/src/main/java/com/kk/app/mobile/SplashActivity.java
package com.kk.app.mobile;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.os.Handler;
import android.view.WindowManager;
import android.widget.ImageView;

import com.bumptech.glide.Glide;
import com.kk.app.mobile.utils.AppJumpUtil;


/**
 * @author kk
 * @datetime: 2020/9/1
 * @des #启动Splash
 */
public class   SplashActivity extends AppCompatActivity {

    private ImageView mImKlcw;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getWindow().addFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_NAVIGATION);
        setContentView(R.layout.app_splash_avy);
        initMain();
    }

    private void initMain() {
        mImKlcw = findViewById(R.id.im_klcw);
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                AppJumpUtil.startMainAvy(SplashActivity.this);
                finish();
            }
        },2200);
    }
}

File: app/src/main/java/com/kk/app/mobile/SplashActivity.kt
package com.kk.app.mobile

import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity
import com.kk.app.mobile.utils.AppJumpUtil

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash_activity)
        initMain()
    }

    private fun initMain() {
        Handler().postDelayed({
            AppJumpUtil.startMainAvy(this@SplashActivity)
            finish()
        }, 1000)
    }
}